src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java

```diff
@@ -168,12 +168,18 @@ public void verifyVideo() {
     verifyComponent(video);
   }
 
-  public void verifyContentLoaded() {
+  public boolean isContentLoaded() {
     wait.forElementVisible(iframe);
     driver.switchTo().frame(iframe);
-    wait.forElementVisible(contentInput);
-    PageObjectLogging.log("verifyContentLoaded", "RTE editor loaded", true, driver);
-    driver.switchTo().defaultContent();
+    try {
+      wait.forElementVisible(contentInput);
+      driver.switchTo().defaultContent();
+      return true;
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)